src/Compilers/CSharp/Portable/Binder/Binder_Query.cs

```diff
@@ -229,10 +229,13 @@ private static BoundCall ReverseLastTwoParameterOrder(BoundCall result)
             argsToParams.AddRange(Enumerable.Range(0, n));
             argsToParams[n - 1] = n - 2;
             argsToParams[n - 2] = n - 1;
+            var defaultArguments = result.DefaultArgumentsOpt.Clone();
+            (defaultArguments[n - 1], defaultArguments[n - 2]) = (defaultArguments[n - 2], defaultArguments[n - 1]);
```

Comment:
- The indexer `get` accessor returns `false` for any argument beyond the capacity of the BitVector, including if the receiver is `default(BitVector)`
- The indexer `set` accessor will grow the capacity of the BitVector internally if needed to be able to write the flag to the vector. We could make this so that the capacity only grows if we assign `true` to the indexer.

I actually didn't think of any scenarios where the absence of this change would cause a problem, I just reviewed the BitVector indexer implementation and more or less imitated what was already here in this method.